aireview/README.md

```diff
@@ -1,2 +1,54 @@
 # AIReview
-Pull request review agent
\ No newline at end of file
+Pull request review agent
```

Comment:
**documentation**: Repository URL placeholder should be replaced with the actual repository URL to make the documentation immediately usable